docker-compose.yml

```diff
@@ -29,7 +29,7 @@ services:
     environment:
      - BASE_URL=https://test-suite.pilot-prep.syncfor.science
     volumes:
-      - tests.db:/usr/src/app/testsuite/db
+      - tests.db:/db
       - ./config/tests/:/usr/src/app/config
```

Comment:
I think my proposed change is actually wrong: 3 slashes means relative and 4 is absolute ?